deploy/docker/server.py

```
@@ -178,6 +197,31 @@ async def add_security_headers(request: Request, call_next):
 }
 
 
+logger = logging.getLogger(__name__)
+
+
+def _resolve_binary_base_dir() -> str:
+    base_dir = config.get("binary", {}).get("default_dir", "/tmp/crawl4ai-exports")
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/crawl4ai-exports"

(S108)